code/mob/living/critter/small_animal.dm

```diff
@@ -2880,7 +2880,7 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 
 /datum/targetable/critter/mentordisappear
 	name = "Vanish"
-	desc = "Leave your body and return to ghost form"
+	desc = "Leave your body and return to the etherial realm."
```

Comment:
?

Reply:
sp.

Reply:
moreso wondering why this is changed at all

Reply:
that was due to some merge conflict fuckiness with my master branch, fixed